royalties-registry/test.sh

```diff
@@ -1,2 +1,11 @@
 #!/usr/bin/env bash
-truffle test
\ No newline at end of file
+truffle test ./test/RoyaltiesRegistry.test.js \
```

Comment:
i don't think we need .sh files for tests if there is only 1 test file

Reply:
ok, delete